src/schema/v2/me/index.ts

```diff
@@ -138,6 +139,20 @@ const Me = new GraphQLObjectType<any, ResolverContext>({
         })
       },
     },
+    unreadConversationCount: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: "The count of conversations with unread messages.",
+      resolve: (_root, _options, { conversationsLoader }) => {
+        if (!conversationsLoader) return 0
+        const { page, size } = convertConnectionArgsToGravityArgs({
```

Comment:
This feels indirect (to derive `page` and `size` from a 'fake' connection-style arg of `first: 1`). Usually this helper is used for connection resolvers which do accept those connection-style arguments (typically from Relay containers).

You can prob leave out `page` (Impulse would default to 1), and `size` of 0 might work better than 1 (does Impulse work w/ that? every little bit helps!).

Reply:
I'll test that out!